<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1">
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1" columnstretch="1,3">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>姓名</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>性别</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="radioButtonMan">
         <property name="text">
          <string>男</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/images/man.png</normaloff>:/images/man.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonWoman">
         <property name="text">
          <string>女</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/images/woman.png</normaloff>:/images/woman.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>性格特质</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
     <item>
      <widget class="QPushButton" name="pushButtonCommit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>提交</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/images/yes.png</normaloff>:/images/yes.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/images/no.png</normaloff>:/images/no.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>pushButtonCancel</zorder>
  <zorder>label</zorder>
  <zorder>lineEditName</zorder>
  <zorder>label_2</zorder>
  <zorder>radioButtonMan</zorder>
  <zorder>radioButtonWoman</zorder>
  <zorder>label_3</zorder>
  <zorder>comboBox</zorder>
  <zorder>pushButtonCommit</zorder>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
